packages/react-server/src/render/render.tsx

```diff
@@ -74,6 +80,15 @@ export function createRender(render: RenderFunction, options: Options = {}) {
         ? assetNameInput(ctx)
         : assetNameInput;
 
+    const {
+      scripts: additionalScripts = [],
+      styles: additionalStyles = [],
+      ...additionalHtmlProps
+    } =
+      typeof htmlPropsInput === 'function'
```

Comment:
A bit nitpicky but I found this a bit difficult to parse at first glance because of the line break. 
Maybe split this up into 2 expressions by pulling out this ternary into a  `const props` or something.

Reply:
I agree, the linebreak after the `=` is quite unconventional. Not sure it's worth adding a level of indirection to fix this, though.